packages/components/src/components/Card/Card.tsx

```diff
@@ -99,6 +101,7 @@ export type CardProps = AccessibilityProps &
         label?: ReactNode;
         paddingType?: PaddingType;
         fillType?: FillType;
+        contentOverflow?: Overflow,
```

Comment:
what is the difference between `contentOverflow` and `overflow` from `AllowedFrameProps`?

Reply:
I rewrote it to overflow, it's cleaner now.